### PR TITLE
add beauty audio-player for all task requirements

### PR DESCRIPTION
**Pull request by dimahalaiskiy**

### CODE REVIEW

**Comment by dimahalaiskiy** [comment]
Uploading Screen Recording 2021-07-31 at 18.17.10.mov…

File: src/app/models/listening-test/listening-test.component.ts [hunk 1]
```diff
@@ -1,10 +1,85 @@
...
+    this.elementRef.nativeElement
+      .querySelector('.play-pause')
+      .addEventListener('click', this.onClick.bind(this));
+  }
+
+  onClick() {
+    this.elementRef.nativeElement.querySelector('.play-pause').classList.add('play-pause-disable');
+    this.counter += 1;
+    if (this.counter === 1) {
+      this.elementRef.nativeElement
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:64]: create variable to store result of query selector to not duplicate same code twice

File: src/app/models/listening-test/listening-test.component.ts [hunk 2]
```diff
@@ -1,15 +1,92 @@
...
+        .querySelector('.play-pause')
+        .classList.remove('play-pause-disable');
+    }
+  }
+
+  onEnded() {
+    this.counter += 1;
+    if (this.counter <= 4) {
+      this.elementRef.nativeElement
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:84]: same here

File: src/app/models/listening-test/listening-test.component.ts [hunk 3]
```diff
@@ -1,15 +1,92 @@
...
+  constructor(private elementRef: ElementRef) {}
   questions: Question[] = [];
 
+  counter = 0;
+
+  msaapAutoPlay = false;
+
+  msaapDisplayTitle = false;
+
+  button = document.querySelector('.play-pause');
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:25]: do you use it?

File: src/app/models/listening-test/listening-test.component.ts [hunk 4]
```diff
@@ -1,15 +1,92 @@
...
+      this.elementRef.nativeElement
+        .querySelector(
+          '#mat-tab-content-0-1 > div > app-listening > ngx-audio-player > mat-card > button.mat-focus-indicator.ngx-p-1.volume.justify-content-center.mat-button.mat-button-base.ng-star-inserted',
+        )
+        .click();
+      this.elementRef.nativeElement
+        .querySelector(
+          '#mat-tab-content-0-1 > div > app-listening > ngx-audio-player > mat-card > button.mat-focus-indicator.ngx-p-1.volume.justify-content-center.mat-button.mat-button-base.ng-star-inserted',
+        )
+        .classList.add('play-repeat-disable');
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:73]: lets disable repeat not on first click, but automaticly

File: src/app/models/listening-test/listening-test.component.ts [hunk 5]
```diff
@@ -1,15 +1,92 @@
...
+
+  msaapDisplayVolumeControls = true;
+
+  msaapDisplayRepeatControls = true;
+
+  msaapDisplayArtist = false;
+
+  msaapDisplayDuration = false;
+
+  msaapDisablePositionSlider = false;
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:39]: Do we want allow user to use audio slider? maybe lets disable it

File: src/app/models/listening-test/listening-test.component.ts [hunk 6]
```diff
@@ -1,15 +1,92 @@
...
+  ];
+
   ngOnInit(): void {
     this.questions = [...MOCK_QUESTION_LIST_AUDITION];
   }
-}
+
+  ngAfterViewInit() {
+    this.elementRef.nativeElement
+      .querySelector('.play-pause')
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:56]: same here

File: src/app/models/listening-test/listening-test.component.ts [hunk 7]
```diff
@@ -1,15 +1,92 @@
...
+        )
+        .click();
+      this.elementRef.nativeElement
+        .querySelector(
+          '#mat-tab-content-0-1 > div > app-listening > ngx-audio-player > mat-card > button.mat-focus-indicator.ngx-p-1.volume.justify-content-center.mat-button.mat-button-base.ng-star-inserted',
+        )
+        .classList.add('play-repeat-disable');
+    } else {
+      this.elementRef.nativeElement
+        .querySelector('.play-pause')
```
Comment (RomPavel) [inline src/app/models/listening-test/listening-test.component.ts:76]: same here

File: src/app/models/listening-test/listening-test.component.ts (hunk 3 above)
Comment (dimahalaiskiy) [reply src/app/models/listening-test/listening-test.component.ts:25]: No, I dont, already deleted

File: src/app/models/listening-test/listening-test.component.ts (hunk 5 above)
Comment (dimahalaiskiy) [reply src/app/models/listening-test/listening-test.component.ts:39]: Oh, yes, I forgot about this description

**Pull request merged**